api/contrib/envoy/extensions/filters/http/http_wasm/v3alpha/wasm.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.http.http_wasm.v3alpha;
+
+import "envoy/extensions/wasm/v3/wasm.proto";
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.filters.http.http_wasm.v3alpha";
+option java_outer_classname = "HttpWasmProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/filters/http/http_wasm/v3alpha";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: HttpWasm]
+// [#extension: envoy.filters.http.http_wasm]
+
+// [#next-free-field: 2]
+message Wasm {
+  // General Plugin configuration.
+  envoy.extensions.wasm.v3.PluginConfig config = 1;
```

Comment:
let's not reuse the shitty existing Wasm API, but rather define from scratch (e.g. root_id, vm_id vm_config plugin_name etc can be removed, plus we should remove the ambiguity the term `vm_config` vs `plugin_config` where should stick to either one of them 

having said that, API conversation is something we can do latter together with others, so this is just a headsup

Reply:
like I mentioned, we don't need to support anything except local file reads for the datasource of Wasm binary (at least in the first PoC).